stub-environment/roles/BCPC-Hadoop-Head-HBase.json

```diff
@@ -3,7 +3,9 @@
   "json_class": "Chef::Role",
   "run_list": [
     "role[Basic]",
+    "recipe[bcpc-hadoop::phoenixqs_hbase_site]",
     "recipe[bcpc-hadoop::hbase_master]",
+    "recipe[bcpc-hadoop::phoenix_query_server]",
```

Comment:
This will install PQS on all HBase head nodes. Is that the intention?

Reply:
Yes